DNN Platform/Library/Entities/Users/UserEventHandlers.cs

```diff
@@ -51,6 +51,7 @@ public void UserDeleted(object sender, UserEventArgs args)
 
         public void UserApproved(object sender, UserEventArgs args)
         {
+            if (!args.SendNotification) return;
             Mail.SendMail(args.User, MessageType.UserRegistrationPublic, PortalSettings.Current);
             DeleteAllNotifications(args.User.UserID);
```

Comment:
You're right. I've added a new commit (https://github.com/janjonas/Dnn.Platform/commit/aa28f0ca6346fa239a63959cc0a295f69d8eb008) to this pull request. I also changed the name of the method to make more clear what the method does.